PeerioTechnologies/peerio-mobile
# feat: e2e test leave room and navigate to chat list [branch ch7326]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/7326/mobile-e2e-tests-when-leaving-channel-navigate-to-chat-list
#### Testing instructions  
The test itself works. But there's a bug where leaving a room crashes the app. I will see if Eren can reproduce that and make a CH task.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
